### PR TITLE
CSCQVAIN-130: Better toolbars on narrow screens

### PR DESCRIPTION
**Pull request by tahme**
Make toolbars fill the available space better and use multiple rows if necessary.

### CODE REVIEW

File: src/schemas/fairdata.bundle.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ export default {
 	ida: {
 		schema: FairdataIdaSchema,
 		ui: FairdataIdaUi,
-		name: "I want to select IDA files", // "Fairdata (IDA)",
+		name: "Select IDA files", // "Fairdata (IDA)",
```
Comment (jppiiroinen) [inline src/schemas/fairdata.bundle.js]: this is out of scope of this issue.

File: src/schemas/fairdata.bundle.js [hunk 2]
```diff
@@ -72,7 +72,7 @@ export default {
 	att: {
 		schema: FairdataAttSchema,
 		ui: FairdataAttUi,
-		name: "I want to link Remote resources", // "Fairdata (ATT)",
+		name: "Link Remote resources", // "Fairdata (ATT)",
```
Comment (jppiiroinen) [inline src/schemas/fairdata.bundle.js]: this is out of scope of this issue.

File: src/views/Editor.vue [hunk 3]
```diff
@@ -20,16 +20,16 @@
 					</b-form-select>
 				</b-input-group>
 
-				<b-input-group size="sm" class="w-25 mx-1" prepend="owner">
+				<b-input-group size="sm" prepend="Owner">
```
Comment (jppiiroinen) [inline src/views/Editor.vue:23]: is the case change from `owner` to `Owner` intended change

File: src/views/Editor.vue [hunk 4]
```diff
@@ -93,8 +93,8 @@
 
 				<p>Where are your files related to this dataset:
 				<ul>
-				<li>In Fairdata IDA (you want to select files from IDA)" and "I want to select Fairdata IDA files"</li>
-				<li>Somewhere else (you want to link files from remote location)</li>
+				<li>In Fairdata IDA (you want to select files from IDA): "Select IDA files"</li>
```
Comment (jppiiroinen) [inline src/views/Editor.vue]: out of scope of this issue

File: src/views/Editor.vue [hunk 5]
```diff
@@ -93,8 +93,8 @@
...
+				<li>Somewhere else (you want to link files from remote location): "Link Remote Resources"</li>
```
Comment (jppiiroinen) [inline src/views/Editor.vue]: out of scope of this issue

**Review by jppiiroinen — CHANGES_REQUESTED** [review]

File: src/views/Editor.vue (hunk 3 above)
Comment (tahme) [reply src/views/Editor.vue:23]: Yes. A bit out of scope but makes the UI a bit more consistent, and I was changing the line anyway.

Comment (jppiiroinen) [reply src/views/Editor.vue:23]: wanted to confirm that this was not an accidental change

File: src/schemas/fairdata.bundle.js (hunk 1 above)
Comment (tahme) [reply src/schemas/fairdata.bundle.js]: I'd say that having an unnecessarily long text in a field affects usability when space is limited, which is why I did this here, but it's not a problem to move these changes to a new ticket. I'll do that.

Now that I think about it, the reason the schemas have a "name" (shown before schema is selected) and "shortName" (shown after schema is selected) is that the previous toolbar layout often didn't show what schema you had selected because there was only enough space to see the first few characters. Now that the toolbars show the selected dropdown option better, maybe shortNames should be removed so the options would have consistent names that don't change. I'll add that to the ticket.

Comment (jppiiroinen) [reply src/schemas/fairdata.bundle.js]: My concern is that this changes the end user experience and the user manuals/guides as well as tests. So based on that it makes sense to track this kind of change in a separate issue and a pull request.

Comment (tahme) [reply src/schemas/fairdata.bundle.js]: That's a good point for changes that affect in the UI. I'll ping Erja in the ticket to make sure the change is ok.

Comment (tahme) [reply src/schemas/fairdata.bundle.js]: Removed the out-of-scope parts, will make a new PR for them.

**Review by jppiiroinen — APPROVED** [review]

**Comment by jppiiroinen** [comment]
Looks good now, there seems to be a merge conflict with Editor.vue file.
